v1.0/openapi3/headers/Content-Length.yaml

```diff
@@ -1,4 +1,4 @@
 required: false
 schema:
   type: integer
-description: "The `Content-Length` header field indicates the anticipated size of the payload body. Only sent if there is a body.\n**Note:** The API supports a maximum size of 5242880 bytes (5 Megabytes)."
+description: "The `Content-Length` header field indicates the anticipated size of the payload body. Only sent if there is a body.**Note:** The API supports a maximum size of 5242880 bytes (5 Megabytes)."
```

Comment:
Perhaps the description could be split into multiple lines like the others?

Reply:
Also, would it be better to separate the note into a new line?

Reply:
done